code/game/atoms.dm

```diff
@@ -77,6 +77,9 @@
 /atom/proc/emp_act(var/severity)
 	return
 
+/atom/proc/emp_act_console(var/severity)
```

Comment:
Actually, @Mwerezak is correct. Not only is having this as an atom proc unnecessary, but I think having this as a proc at all is slightly unnecessary. The /obj/mecha/emp_act definition can simply be changed to do what the other proc does. Although. This will also increase the lethality that emps otherwise have on mechs, which might not be good.

Easiest and the least conflicting solution would be to simply adjust the call it runs through the console to have a small number.